src/dashboard/src/main/migrations/0075_userprofile_system_emails.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("main", "0074_version_number")]
+
+    operations = [
+        migrations.AddField(
+            model_name="userprofile",
+            name="system_emails",
+            field=models.BooleanField(default=False),
```

Comment:
I'm not sure what's the right approach when the setting is `False` by default here in the migration and in the new model field, but the client scripts are looking for `userprofile__system_emails=True`. Existing users would stop receiving emails, right? @sevein do you have any recommendation on this? Sorry if I'm missing something.

Reply:
Good catch, Douglas. To ensure a seamless upgrade, perhaps we could add a new data migration that enables the setting for existing users.

Reply:
How about just flipping the default to `True`? That means people will have to opt out of emails rather than opting in, but I'm not sure that's actually any worse...

Reply:
CC @sromkey

Reply:
That sounds fine to me, since the current situation is everyone gets the emails by default.